students/rgpag/lesson03/slicingLab.py

```diff
@@ -0,0 +1,52 @@
+def exchange_first_last(seq):
+	""" return seq with first and last elements exchanged"""
+	L=len(seq)
+	return seq[-1]+seq[1:L-1]+seq[0]
+
+	a_string = "this is a string"
+	a_tuple = (1,2,3,4,5,6)
```

Comment:
This code is unreachable because it's inside the function block after the `return` statement, you want to place your assert statements outside of the function at the global module level. You'll see a failing assert for the error I mentioned above